spec/fixtures/state_file/fed_return_jsons/2023/nj/blind_primary.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
[DUST] Doesn't have to be necessarily part of this PR, but I'm curious how we plan on generating these json files moving forward. Is part of the PR strategy to add to these test JSON files every time we introduce a new feature that requires reading a value from JSON? Is there a way to get these JSON files directly from DF instead of needing to construct them ourselves? As is, it feels like we're making these up based on what we think is necessary for our logic, which seems backwards.

Reply:
Agree that manual construction is annoying. These represent data we are getting from df, but that just isn't in the federal return in a way we can predictably access or parse

Reply:
Question for CFA: How do you deal with this? Do y'all just manually construct each time too?

Reply:
It looks like they've got a similar setup for Maryland but I'd be curious to hear too!

Reply:
Yes, right now we manually create the json test data with values that match the xml data when applicable. The expectation is that early next month we will have real test data once we are able to generate it from DF submissions.